tests/minimal_kex_oqs.c

```diff
@@ -17,127 +17,12 @@ void cleanup(uint8_t *alice_msg, size_t alice_msg_len, uint8_t *alice_key,
              uint8_t *bob_key, size_t bob_key_len, void *alice_priv,
              OQS_KEX *kex, OQS_RAND *rnd);
 
-#ifdef ENABLE_KEX_NTRU
```

Comment:
We should remove minimal_kex_oqs.c

Reply:
I'll remove it in issue: https://github.com/open-quantum-safe/liboqs/issues/372